docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,790 @@
+* Adds Control group search settings {kibana-pull}128090[#128090]
+* Adds hierarchical chaining setting to Controls {kibana-pull}126649[#126649]
+* Adds options list API and validation system {kibana-pull}123889[#123889]
+
+Discover::
+Adds `Copy to clipboard` ability for column name of Document Explorer {kibana-pull}123892[#123892]
+
+Elastic Security::
+For the Elastic Security 8.2.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Lens & Visualizations::
+* Adds manual annotations in *Lens* {kibana-pull}126456[#126456]
+* Adds multi-field group by in *TSVB* {kibana-pull}126015[#126015]
+* Adds ability to navigate to *Lens* with your current *TSVB* configuration {kibana-pull}114794[#114794]
+
+Machine Learning::
+* Anomaly Detection: Add link to maps in charts section of Anomaly Explorer {kibana-pull}128697[#128697]
+* Testing trained models in UI {kibana-pull}128359[#128359]
+* Space aware trained models {kibana-pull}123487[#123487]
+
+Management::
+* Adds support for auto-complete for data streams {kibana-pull}126235[#126235]
+* Adds ability to filter Data View UI for runtime fields {kibana-pull}124114[#124114]
+* Adds ability to share data views across spaces via data view management {kibana-pull}123991[#123991]
+
+Observability::
+* Adds button which allows users to signup for the Synthetics service public beta {kibana-pull}128798[#128798]
+* Adds "View in App URL" {{context.viewInAppUrl}} variable to the rule templating language {kibana-pull}128281[#128281]
+* Adds "View in App URL" {{context.viewInAppUrl}} variable to the rule templating language {kibana-pull}128243[#128243]
+* Adds "View in App URL" {{context.viewInAppUrl}} variable to the rule templating language {kibana-pull}127890[#127890]
+* Adds view in app url as an action variable in the alert message for uptime app {kibana-pull}127478[#127478]
+
+For more information about the features introduced in 8.2.0, refer to <<whats-new,What's new in 8.2>>.
+
+[[enhancements-and-bug-fixes-v8.2.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.2.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.2.0]]
+=== Enhancements
+Alerting::
+* Adds error logs in rule details page {kibana-pull}128925[#128925]
+* Simplify error banner on rules {kibana-pull}128705[#128705]
+* Adds Previous Snooze button {kibana-pull}128539[#128539]
+* Adds Snooze UI and Unsnooze API {kibana-pull}128214[#128214]
+* Adds aggs to know how many rules are snoozed {kibana-pull}128212[#128212]
+* Adds a connector for xMatters {kibana-pull}122357[#122357]
+
+Dashboard::
+* Adds option to open dashboard drilldowns in new tab or window {kibana-pull}125773[#125773]
+* Adds range slider Control {kibana-pull}125584[#125584]
+
+Discover::
+* Adds ability to edit histogram as vis {kibana-pull}125705[#125705]
+* Enables document explorer as default {kibana-pull}125485[#125485]
```

Comment:
I wonder if this should be highlighted in a feature, Document explorer has been around for a while, but enabling it as default document table is a huge change